capstone/src/main/webapp/script.js

```diff
@@ -32,6 +35,10 @@ function getRestaurants() {
   });
 }
 
+function searchRestaurants() {
```

Comment:
Is this function used for anything (or will it be)?

Reply:
Oh! Sorry. I forgot to erase that.